src/diagnostics/tracer_diagnostics.jl

```diff
@@ -0,0 +1,38 @@
+# This file is included in Diagnostics.jl
+
+# Tracers
+
+###
+# Downwelling shortwave radiation (3d)
+###
+compute_mmrso4!(out, state, cache, time) =
+    compute_mmrso4!(out, state, cache, time, cache.tracers)
+
+function compute_mmrso4!(
+    out,
+    state,
+    cache,
+    time,
+    tracers,
+)
+    if !(:prescribed_aerosols_field in propertynames(tracers))
+        error("Aerosols do not exist in the model")
+    elseif !(:SO4 in propertynames(tracers.prescribed_aerosols_field))
+        error("SO4 does not exist in the model")
```

Comment:
```suggestion
    :prescribed_aerosols_field in propertynames(tracers) || error("Aerosols do not exist in the model")
    :SO4 in propertynames(tracers.prescribed_aerosols_field) || error("SO4 does not exist in the model")
```

Reply:
Instead of a single if-else, you can have a first step where you check consisntency, and then the rest of the function assumes that you are in a good place